doc/configuration.rst

```diff
@@ -1025,6 +1025,27 @@ resource available on a remote computer
 Used by:
   - `FlashScriptDriver`_
 
+Dediprog
+~~~~~~~~
+A Dediprog resource is used to configure the parameters to a locally installed
+dpmcd instance. It is assumed that dpcmd is installed on the host and the
+executable is configured in:
+
+.. code-block:: yaml
+
+  tools:
+    dpcmd: '/usr/sbin/dpcmd'
```

Comment:
This tools entry belongs to the `DediprogDriver`, since it uses the tool. The example here should be something like:

```
Dediprog:
  vcc: 0
```

Also maybe we should rename this from `Dediprog` to `DediprogFlasher` and the Driver to `DediprogFlashDriver` since dediprog also builds eMMC programmers.

Reply:
While I did add a `vcc` example usage, I think that the tool really belongs here. This is similar to `Flashrom`, as where the tool is located can vary among exporters, so it's a resource attribute. Or did I understand `Flashrom` case wrong?

Reply:
You're right. I'm now wondering how we handle this on the exporter side, but thats not relevant for this PR.